src/modules/experts/styles/ExpertBlock.style.ts

```diff
@@ -0,0 +1,42 @@
+import { makeStyles } from '@material-ui/core';
+
+export const useStyles = makeStyles(() => ({
+  item_0: {
+    gridArea: '1 / 1 / 5 / 2',
+  },
+  item_1: {
+    gridArea: '1 / 2 / 3 / 3',
+  },
+  item_2: {
+    gridArea: '3 / 2 / 5 / 3',
+  },
+  item_3: {
+    gridArea: '1 / 3 / 5 / 4',
+  },
+  item_4: {
+    gridArea: '1 / 4 / 2 / 5',
+  },
+  item_5: {
+    gridArea: '2 / 4 / 3 / 5',
+  },
+  item_6: {
+    gridArea: '3 / 4 / 4 / 5',
+  },
+  item_7: {
+    gridArea: '4 / 4 / 5 / 5',
+  },
+  item_8: {
+    gridArea: '1 / 5 / 3 / 6',
+  },
+  item_9: {
+    gridArea: '3 / 5 / 5 / 6',
+  },
+  item_10: {
+    gridArea: '1 / 6 / 5 / 7',
+  },
```

Comment:
these should not be set in the `ExpertBlock`.

pass the styles from parent grid component, it should be responsible for styling it's children.
item doesn't have to know about it's position, it has to display the content, and that's it.
parent should be responsible for positioning its items.

Reply:
moved to MainExpertsView block